### ISSUE TITLE
Deprecated method showing up twice in description

### ISSUE THREAD

**Issue by mgardiner** [stoicflame/enunciate#104]
Below is an example of a resource with one method that is deprecated.  Note the @deprecated javadoc defines "test" but in the attached screenshot you will notice that "test" is duplicated

```
@Path("myresource")
public class MyResource {

    /**
     * Get some stuff
     * @deprecated test
     * @return Returns Stuff
     */
    @GET
    @Deprecated
    @Produces(MediaType.APPLICATION_JSON)
    public Stuff getIt() {
        Stuff stuff = new Stuff();
        stuff.setFoo("bar");

        return stuff;
    }

    /**
     * Get some stuff
     * @return Returns Stuff
     */
    @GET
    @Produces(Stuff.MEDIA_TYPE)
    public Stuff getStuff() {
        Stuff stuff = new Stuff();
        stuff.setFoo("bar");
        return stuff;
    }
}
```

![sample](https://cloud.githubusercontent.com/assets/1128232/9277532/f1de2178-4267-11e5-8bf8-d956e219aabe.png)

**Comment by stoicflame** [comment]
Thanks for the report.

**Comment by mgardiner** [comment]
You're welcome, and it's looking good.  Keep up the great work!

**Comment by stoicflame** [comment]
Fixed at e61de3c, and released with Milestone 2.
